specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-01-15/cosmos-db.json

```diff
@@ -0,0 +1,8572 @@
+          "description": "The failover priority of the region. A failover priority of 0 indicates a write region. The maximum value for a failover priority = (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists."
+        }
+      }
+    },
+    "RegionForOnlineOffline": {
+      "properties": {
+        "region": {
+          "type": "string",
+          "description": "Cosmos DB region, with spaces between words and each word capitalized."
+        }
+      },
+      "required": [
+        "region"
+      ],
+      "description": "Cosmos DB region to online or offline."
+    },
+    "Location": {
+      "description": "A region in which the Azure Cosmos DB database account is deployed.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The unique identifier of the region within the database account. Example: &lt;accountName&gt;-&lt;locationName&gt;."
+        },
+        "locationName": {
+          "type": "string",
+          "description": "The name of the region."
+        },
+        "documentEndpoint": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The connection endpoint for the specific region. Example: https://&lt;accountName&gt;-&lt;locationName&gt;.documents.azure.com:443/"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        },
+        "failoverPriority": {
+          "description": "The failover priority of the region. A failover priority of 0 indicates a write region. The maximum value for a failover priority = (total number of regions - 1). Failover priority values must be unique for each of the regions in which the database account exists.",
+          "format": "int32",
+          "type": "integer",
+          "minimum": 0
+        },
+        "isZoneRedundant": {
+          "type": "boolean",
+          "description": "Flag to indicate whether or not this region is an AvailabilityZone region"
+        }
+      }
+    },
+    "ARMResourceProperties": {
+      "type": "object",
+      "description": "The core properties of ARM resources.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource group to which the resource belongs."
+        },
+        "tags": {
+          "$ref": "#/definitions/Tags"
+        },
+        "identity": {
```

Comment:
All of the resources utilizing this top level definition actualyl support managed identity?

Reply:
I believe it is only supported by our top level resource, DatabaseAccount. It should not be added in other places. @xujin-zhang please fix it.

Reply:
Fix has been made!